studio-docs/source/sharing-graphs.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Sharing your graph
+---
+
+import {
+  ExpansionPanel,
+} from 'gatsby-theme-apollo-docs/src/components/expansion-panel';
+
+Apollo Studio provides features that help you share your graph with both teammates and external consumers.
+
+
+## Making a graph public
+
+You can make individual [variants](./org/graphs/#managing-variants) of your graph public. If you do, people _outside_ your organization can view the following pages for that variant in Studio:
+
+* Home
+* Schema
+* Explorer
+
+This enables consumers of your graph to learn about your graph's schema and any special usage information (such as authentication details) that you've documented in [the variant's README](#customizing-a-graphs-homepage). They can also run operations against your graph with the Explorer.
+
+People outside your organization can't view any _other_ pages for a public variant (Fields, Operations, etc.), and they can never view _any_ pages for private variants. New variants always start as private.
+
+> **Note:** Only organization members with the [`Org Admin` or `Graph Admin` role](./org/members/) can set a variant's visibility.
+
+**To make a variant public:**
```

Comment:
@StephenBarlow Capability to make a variant public or not is only available to org/graph admins.